packages/truffle-core/test/mnemonic.js

```diff
@@ -0,0 +1,33 @@
+const assert = require("chai").assert;
+const accountsInfo = require("../lib/mnemonics/mnemonic");
+const Config = require('truffle-config');
+const defaultUserConfig = Config.getUserConfig();
+
+
+describe("mnemonic", function() {
+  describe("#getOrGenerateMnemonic", function(){
+    it("checks user-level configuration for mnemonic and creates one if one is not present", function(done) {
+      this.timeout(30000);
```

Comment:
What is the timeout for?

Reply:
I saw other tests in the repo adding a timeout longer than the default (I think 2000?), so I added it in to be safe. Do you think I should remove it?